### ISSUE TITLE
Poor downloading - migration to webtorrent

### ISSUE THREAD

**Issue by aurimus** [PopcornTime-CE/desktop#78]
Any reason why same torrent with many seeders would be downloading 10x faster on Transmission than on Popcorn? It seems just randomly slow or fast, but 9 out of 10 - slow.

**Comment by YIFYtorrents** [comment]
I honestly have no idea why that is slower than transmission.

**Comment by pRieStaKos** [comment]
I confirm this issue exists

**Comment by aurimus** [comment]
usually very small number of connected peers

**Comment by YIFYtorrents** [comment]
use google cloud if you want faster downloads

**Comment by aurimus** [comment]
Its terrible quality if yoi havent noticed

**Comment by YIFYtorrents** [comment]
not anymore read this https://www.reddit.com/r/PopcornTimeCE/comments/4nq107/api_update_google_cloud_now_supports_1080p_quality/

The quality has improved dramatically.

**Comment by muzena** [comment]
@YIFYtorrents 
Google cloud don't work on linux

**Comment by YIFYtorrents** [comment]
@muzena 

> # Error
> 
> The video could not be loaded, either because the server or network failed or because the format is not supported
> 
> Go to your popcorntime directory desktop and paste/replace [libffmpegsumo.so](http://www33.zippyshare.com/v/4UZX7pu4/file.html) in it.

Source:  https://github.com/PopcornTimeCommunity/desktop/blob/master/README.md#error

**Comment by muzena** [comment]
@YIFYtorrents 
Yes, I  replaced that but still don't work!

**Comment by vzamanillo** [comment]
I am plannig a migration to webtorrent in my forked repo.

**Comment by YIFYtorrents** [comment]
> I am plannig a migration to webtorrent in my forked repo.

that would be great, Do you have a link to your repo?

**Comment by vzamanillo** [comment]
Yes but will be dead soon, I will change to official popcorn and move all my changes / enhancements on because is still a community.

**Comment by pRieStaKos** [comment]
I'm using PopcornTime CE to watch a tv episode and some of them having poor connections or few peers. But when I click the magnet and download the episode through torrent client, peers are not so few and download speed is as expected.

You should check it why this is happenging. Is there a limit gap on peers or speed ?

**Comment by andresespinosapc** [comment]
@vzamanillo what do you mean with changing to official popcorn?

**Comment by vzamanillo** [comment]
The official PCT is working very hard migrating to new NW.js version fixing all issues to release an improved app. This fork is dead and is not a community, you can do a request, PRs, anything... No response. I don't know why the people is still waiting anything here.

**Comment by YIFYtorrents** [comment]
@vzamanillo all PRs are merged. Matter in fact, this fork is still alive and kicking. Check out the latest commits and changelog v4.1-5. The latest version has a plugin system, rarbg search, html5 video player, etc. new filter opotions, etc. check it out and think twice before recommending a fork that is run by scammers / MPAA. http://popcorntime.ag/official-statement.html#Malware2
